src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/HMACSHA1.cs

```diff
@@ -3,14 +3,15 @@
 
 using Internal.Cryptography;
 using System.ComponentModel;
+using System.Runtime.Versioning;
 
 namespace System.Security.Cryptography
 {
     //
     // If you change anything in this class, you must make the same change in the other HMAC* classes. This is a pain but given that the
     // preexisting contract from the .NET Framework locks all of these into deriving directly from HMAC, it can't be helped.
     //
-
+    [UnsupportedOSPlatform("browser")]
```

Comment:
Please restore the blank lines between the meta-comment and the code for all the HMAC classes.

Reply:
Done